raiden/network/proxies/token_network.py

```diff
@@ -290,7 +288,7 @@ def _new_netting_channel(
                 required_gas=self.gas_measurements["TokenNetwork.openChannel"],
                 block_identifier=checking_block,
```

Comment:
The checking block was queried before the gas estimation and the precondition checks, this may be too old. Please move it inside this `if not gas_limit` so that it is closest to the latest block.

Reply:
I don't understand, the value is still `"latest"` or `"pending"`?

Reply:
`latest` and `pending` are not real blocks, and they change under our feet, please refer to https://github.com/raiden-network/raiden/issues/2885 for more details

Reply:
Ok, using a fixed blockhash for the insufficient eth and postcondition checks now.